config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: myflix_development
+  pool: 5
+  username: boottest
+  password: Iputo*34

```

Comment:
Thanks @alagram.
You are right. Didnt consider it was a problem, I suppose the solution is including it in the gitignore file. is that right?. 
By the way, what about deploying to heroku?. Do you have to include a production section in the file? Are the databases automatically created or you need to include database name, user and password?